OxfordBodleian/Juel-Jensen/BDLaethf29.xml

```diff
@@ -33,26 +34,337 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    rend="rubric">ሥላሴ፡</hi> ኅቡር፡ ተማኅፀንኩ፡ ወአማኅፀንኩ፡ ነፍስየ፡ ወሥጋየ፡ ከመ፡
+                                ይምሐረኒ፡ ወይሣሃለኒ፡ በእንተ፡ ቅዱስ፡ <hi rend="rubric">ሥላሴ</hi>ሁ፡ ሊተ፡ ገብሩ፡
+                                    <persName ref="PRS13017GabraMar"><subst>
+                                        <del rend="erasure"/>
+                                        <add place="overstrike">ገብረ፡</add>
+                                    </subst>
+                                    <hi rend="rubric">ማርያም፡</hi></persName> ለዓለመ፡ ዓለም፡ አሜን፡ ወአሜን፡
+                                ለይ፡ ለ። </explicit>
+                        </msItem>
+
+                        <msItem xml:id="ms_i1.5" corresp="#h1">
+                            <locus from="45v" to="55r"/>
+                            <title type="complete" ref="LIT2296Sayfas#Thursday"/>
+                            <incipit xml:lang="gez">
+                                <pb n="45v"/>
+                                <locus target="#45v"/>
+                                <hi rend="rubric">አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ በግፃዌ፡ ሥሉስ፡</hi> በሥላሴ፡ ግጽው፡
+                                ዋህድ፡ አምላክ፡ ፫አዕማደ፡ <hi rend="rubric">ብርሃን፡ ፩አምጣኖሙ፡ ወ፩ስኖሙ፡
+                                ፫መባርቅ</hi>ት፡ </incipit>
+                            <explicit xml:lang="gez">
+                                <locus target="#55r"/>በዝንቱ፡ ስመ፡ <hi rend="rubric">ሥላሴ፡</hi> ተማኅፀንኩ፡
+                                ከመ፡ ይክፍለኒ፡ <hi rend="rubric">ሥላሴ፡</hi> ቅዱስ፡ እምተስፋ፡ ቅዱሳን፡ እንተ፡ ይእቲ፡
+                                ደብተራ፡ ጽድቅ፡ ጽዮን፡ ሊተ፡ ገብርከ፡ <persName ref="PRS13017GabraMar"><subst>
+                                        <del rend="erasure"/>
+                                        <add place="overstrike">ገብረ፡</add>
+                                    </subst>
+                                    <hi rend="rubric">ማርያም፡</hi></persName>
+                            </explicit>
+                        </msItem>
+
+                        <msItem xml:id="ms_i1.6" corresp="#h1">
+                            <locus from="55v" to="66r"/>
+                            <title type="complete" ref="LIT2296Sayfas#Friday"/>
+                            <incipit xml:lang="gez">
+                                <pb n="55v"/>
+                                <locus target="#55v"/>
+                                <hi rend="rubric">ዓይ፡ ልሳን፡ ወዓይ፡ ከናፍር፡ ዘይክል፡ ነቢበ፡</hi> ዕበየ፡ ክብሩ፡
+                                ለሥላሴ፡ ዓይ፡ ልቡና፡ ወዓይ፡ ሰሚዕ፡ ዘይክል፡ አእሜ<hi rend="rubric">ሮ፡ ህላዌሆሙ፡
+                                    ለሥላሴ።</hi></incipit>
+                            <explicit xml:lang="gez">
+                                <locus from="65v" to="66r"/><hi rend="rubric"><sic>ስብሐ፡</sic></hi>
+                                ለእግዚአብሔር፡ ሰብእ፡ ወመላእክት፡ <pb n="66r"/> ኢየአምርዎ፡ ለምክሩ፡ ወለምሥጢሩ። ለዓለመ፡
+                                ዓለም፡ አሜን። ወአሜን፡ ለይኵን፡ ለይኩን። ዕቀበኒ፡ ወአድኅነኒ፡ እመከራ፡ ሥጋ፡ ወነፍስ፡ ሊተ፡ ለገብርከ፡
```

Comment:
It is completely optional from the point of view of the encoding (as long as we don't want to somehow mark supplications in the catalouges, which I think we don't). I usually do it because I find the subscriptions, supplications etc. interesting, but there is no necessity. 
(and now I think that my way in the Taʾamməra Māryām manuscript was, while fine from the encoding, maybe not the most economic. There could be one `<explicit>` in which the supplication is marked up with `<seg type="supplication">`. But, again, from the encoding both are fine and not marking up supplications is also fine :) )